src/forwarders.js

```diff
@@ -589,33 +604,78 @@ export default function Forwarders(mpInstance, kitBlocker) {
                 'No config was passed. Cannot process forwarders'
             );
         } else {
-            try {
-                if (
-                    Array.isArray(config.kitConfigs) &&
-                    config.kitConfigs.length
-                ) {
-                    config.kitConfigs.forEach(function(kitConfig) {
-                        self.configureForwarder(kitConfig);
-                    });
-                }
+            this.processMpConfiguredKits(config);
```

Comment:
I think we need a short comment to explain this process, ideally to explain briefly the flow so that we can remember next time we refactor. It should either go here, or maybe in the function that calls `processForwarders`. Your choice.